ecf/scripts/plots/cam/jevs_cam_href_grid2obs_ecnt_past31days_plots.ecf

```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:15:00
-#PBS -l place=vscatter:shared,select=1:ncpus=72:mem=100GB
+#PBS -l place=vscatter:shared,select=2:ncpus=66:mem=500GB
```

Comment:
@malloryprow This .ecf script line includes` shared`, but this is not included in the dev driver version. Do we know which version is correct? Can the dev drivers and .ecf versions match? Thanks!

Reply:
/lfs/h1/ops/prod/packages/evs.v1.0.13/ecf/scripts/plots/cam/jevs_cam_href_grid2obs_ecnt_past31days_plots.ecf has `#PBS -l place=vscatter:shared,select=2:ncpus=66:mem=500GB` so we will go with shared since that is what NCO implemented.